meta/documents/user_guide/de/settings_guide.md

```diff
@@ -0,0 +1,175 @@
+
+### Standardversandanbieter **(nicht genutzt)**
+Die Einstellung `Standardversandanbieter` ist eine Legacy-Einstellung, die das Verhalten des Plugins nicht mehr beeinflusst.
+**Wenn diese Einstellung in Ihrem System angezeigt wird, empfiehlt Wayfair dringend, dass Sie Ihr Plugin auf eine neuere Version aktualisieren.**
+
+### Item Mapping Method
+Die Einstellung `Item Mapping Method` bestimmt das Verhalten beim Abgleichen der Artikelvariationen von Plentymarkets mit den Wayfair-Produkten. Es wird verwendet, wenn die Bestandslisten an Wayfair gesendet werden und um die angeforderten Produkte in einer Wayfair-Bestellung auszuwählen. Der Benutzer von Plentymarkets sollte diese Einstellung so konfigurieren, dass sie der Art und Weise entspricht, in der die Elementvariationen ausgefüllt werden.
+
+Standardmäßig wird das Feld `Variationsnummer` für die Zuordnung der Artikel von Wayfair-Bestellungen zu Artikelvariationen von Plentymarkets verwendet.
+
+![variation number field](../../../images/de/variation_number_field.png)
+
+Die anderen Optionen sind `EAN` (Barcode) und `SKU`.
+
+#### Verwenden von EAN (Barcode) als Elementzuordnungsmethode
+Wenn die `Item Mapping Method` auf `EAN` eingestellt ist, sollte jede Artikelvariation in Plentymarkets so eingerichtet werden, dass sie einen Barcode enthält, der die Wayfair-Lieferanten-Teilenummer widerspiegelt, die Wayfair in eingehenden Bestelldaten sendet:
+
+![barcode field](../../../images/de/variation_barcode_field.png)
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >>` Artikel bearbeiten`
+
+2. Suchen Sie nach Artikeln, die auf Wayfair verkauft werden sollen
+
+3. **Klicken Sie für jeden Artikel** auf den Artikel in den Suchergebnissen und dann auf `Varianten` ![plus button](../../../images/common/button_plus.png)
+
+4. **Für jede der Varianten**:
+    1. Klicken Sie auf die Registerkarte `Einstellungen`
+    2. Wählen Sie im Abschnitt "Barcode" einen Barcode-Typ aus, klicken Sie auf die Schaltfläche `Hinzufügen` und geben Sie den Barcode-Wert in das Feld `1`Code` ein. Der Barcode sollte mit der Wayfair-Lieferanten-Teilenummer übereinstimmen.
+    3. Klicken Sie auf der Variantenstufe auf die Schaltfläche `Speichern` ![save button](../../../images/common/button_save.png) (nicht zu verwechseln mit der Schaltfläche `Speichern` für den Artikel, einige Zeilen darüber).
+
+#### Verwenden der SKU als Elementzuordnungsmethode
+Wenn die "Artikelzuordnungsmethode" auf "SKU" eingestellt ist, sollte jede Artikelvariation in Plentymarkets so eingerichtet werden, dass sie eine Wayfair-spezifische SKU enthält, die die Wayfair-Lieferanten-Teilenummer widerspiegelt, die Wayfair in eingehenden Bestelldaten sendet:
+
+![sku field](../../../images/de/variation_sku_field.png)

```

Comment:
This screenshot does not have the Zwiesitzer Amsterdam at Dawn blurred like in the English screenshot